tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Http/WebRequest/WebRequestCommon.cs

```diff
@@ -23,7 +23,6 @@ internal class WebRequestCommon
 
         internal const string Major2 = "2";
         internal const string Major4 = "4";
-        internal const string Major8 = "8";
```

Comment:
Was this not used?

Reply:
It was, but it isn't any more, as using `SupportedVersions.SupportedDotNet` instead 🙂